code/game/gamemodes/cm_process.dm

```diff
@@ -170,6 +170,41 @@ dat += " You failed to evacuate \the [MAIN_SHIP_NAME]"
 		to_chat(world, dat)
 
 
+/datum/game_mode/proc/declare_completion_announce_round_stats()
+	set waitfor = 0
+	sleep(140)
+	for(var/mob/player in player_list)
+		if(player?.client?.prefs?.toggles_chat & CHAT_STATISTICS)
+			to_chat(player, "<span class='round_body'>The end of round statistics are:</span>")
+			to_chat(player, " ")//gotta get that formatting on.
+			to_chat(player, "There were [round_statistics.total_bullets_fired] total bullets fired.")
+			to_chat(player, "[round_statistics.total_bullet_hits_on_humans] bullets managed to hit marines. For a [(round_statistics.total_bullet_hits_on_humans / max(round_statistics.total_bullets_fired, 1)) * 100]% friendly fire rate!")
```

Comment:
Are these hits on humans or hit on marines? You can add a FF check on the same place where the FF logs are generated.

Reply:
These are hits on humans, this check happens in the projectiles.dm